application/views/dashboard/index.php

```diff
@@ -44,6 +44,17 @@ function echo_table_col($row, $name) {
 		case 'Name': echo '<td>' . ($row->COL_NAME) . '</td>'; break;
 	}
 }
+//Note to reviewer: This function is used a few times in the different views
+//Not sure if it can be moved to a central file to only be used once
+
+//Converts UTC time to local time using PHP's timezones.
+//This accounts for Daylight Savings (apparently)
+function UTCtoLocal($timestamp, $timezone){
+	$utc_date = (new DateTime('@' . $timestamp))->setTimezone(new DateTimeZone('UTC'));
+	$local_date = $utc_date;
+	$local_date->setTimeZone(new DateTimeZone($timezone));
+	return $local_date;
+}
```

Comment:
You are right. All these function you placed in the views shouldn't be there. You could place a function like this in the Library GenFunctions and call it in the Controller to pass the already calculated time to the view. 

Views always should only display data. I know we have some functions and PHP code in the views aswell but some are leftovers and need a refactor and others are necessary. Whenever possible we should stick to the MVC Pattern

Models - Database Actions and "I do/calculate something"
Views - Display the data
Controllers - Decide which data to gets display. 

-> Librarys are used for specific things and could be used here